### PR TITLE
LRAC-8396 | Master

### PR DESCRIPTION
**Pull request by Hanlf**
https://issues.liferay.com/browse/LRAC-8396

For the commit([Use select to choose date](https://github.com/liferay-qa/liferay-portal/commit/e2395758993b9fb517f5d912685c5e93c50f83bb)), it will fix the failures like [ActiveIndividuals#CanSetDataWithCustomRangeOnIndividualActivitiesCard](https://testray.liferay.com/zh/home/-/testray/case_results/1552974877)

### CODE REVIEW

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @Hanlf , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTimeFilter.macro [hunk 1]
```diff
@@ -58,23 +58,27 @@ definition {
...
-		Click(locator1 = "ACTimeFilter#DATE_PICKER_MONTH_DROPDOWN_OPTION");
+		Select(
+			locator1 = "ACTimeFilter#DATE_PICKER_MONTH_DROPDOWN",
+			value1 = "${key_month}");
 
-		Click(locator1 = "ACTimeFilter#DATE_PICKER_YEAR_DROPDOWN");
-		Click(locator1 = "ACTimeFilter#DATE_PICKER_YEAR_DROPDOWN_OPTION");
+		Select(
+			locator1 = "ACTimeFilter#DATE_PICKER_YEAR_DROPDOWN",
+			value1 = "${key_year}");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTimeFilter.macro:67]: Nice @Hanlf ! Thanks for this solution! 👏🏼

**Pull request closed without merging**